compiler/src/dotty/tools/dotc/core/Denotations.scala

```diff
@@ -791,7 +791,7 @@ object Denotations {
       val currentPeriod = ctx.period
       val valid = myValidFor
 
-      def signalError() = println(s"error while transforming $this")
+      def signalError() = () //println(s"error while transforming $this")
```

Comment:
Since that method is only used in one place I would just replace its call by the commented out println

Reply:
Why not use one of the Printers?

Reply:
There isn't a good Printer to use here. Nearby code just uses println.